R/api.R

```diff
@@ -128,11 +135,14 @@ fetch_problem <- function(slug, track_id = "r", force = FALSE, open = TRUE) {
 #' }
 #' @export
 fetch_next <- function(track_id = "r", skip = FALSE,
-                       force = FALSE, open = TRUE) {
+                       force = FALSE, open = TRUE,
+                       start_testing = FALSE) {
 
+  slug <- check_next_problem(track_id = track_id)
```

Comment:
Assigning a value for `slug` here isn't really useful - see the below comment on line 175.

Reply:
True, but removing also removes it for the if (skip) section. 🤔 I'll try moving the slug assignment inside that section.